packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -218,7 +220,8 @@ export const createDefaultCallingHandlers = memoizeOne(
         return;
       }
 
-      await callClient.createView(call.id, undefined, localStream, options);
+      const { view } = (await callClient.createView(call.id, undefined, localStream, options)) ?? {};
```

Comment:
I want Rust's `Option<T>.map(|value| Ok(value)` (Look inside this optional thing, if you find it modify the value using this closure and return back an option) :)